test/quantization/test_quantize_script.py

```diff
@@ -1267,119 +1269,310 @@ def forward(self, x):
         checker.run(model.graph)
         checker.run(model_functional.graph)
 
+    @unittest.skipUnless('fbgemm' in torch.backends.quantized.supported_engines,
+                         " Quantized operations require FBGEMM. FBGEMM is only optimized for CPUs"
+                         " with instruction set support avx2 or newer.")
     def test_quantized_add(self):
-        class Add(torch.nn.Module):
+        class QuantizedAdd(torch.nn.Module):
```

Comment:
For brevity, you could reuse all the tests for add for mul, if you make the operator itself a parameter when you define the model.

Reply:
that would be ideal if we can do that... I don't know if it will work with torchscript if we make operator as an argument, but we may have a refactor later